notes/transcript11.md

```diff
@@ -0,0 +1,585 @@
+
+Ricardo: I think most things can be written by themselves. But I think some simple things that are only taken for convenience, that if you take like 100 lines to write it, that of course should not be a dependency. But if something very large, also something about the trust, if we do not trust the dependency, we can audit that specific version and get that commit hash of that dependency and lock there. There is this two cases that let's see.
+
+# Fee Structure
+
+28:00 - 29:00
+
+Pedro: If there's no further questions I think that's it.
+
+Oskar: Cool. Awesome. There's been a bit of discussion for fee structures for SNT Utility. Rachel do you wanna give a brief overview?
+
+Rachel: So this is coming up in context of Sticker Market sales because we're doing work to finalize the Sticker Market registry contact right now. Specifically whether we should take a percentage of each sale to fund our project directly or to burn perhaps. I guess the heart of the matter is really about how our project sustains itself financially in the long run.
+
+29:00 - 30:00
+
+Rachel: The predominant hypothesis seems to be that token velocity and value combined with having good open source contributions should eliminate the need for us to ever be charging rent for features but there might be worth burning a percentage of each sale of stickers in the meantime to have a deflationary effect. So I'm not sure if we actually want to get into the debate about these structures overall and financial sustainability of the the project or if they should just stick to the focus of the sticker market registry contract and regarding the fees for that right now, what decision do we take for the contract and how do we future-proof it for any like potentially desired changes in the future.
+
+30:00 - 31:00
+
+Ricardo: I think if we have something like the sticker markets and the Croatian markets then having this operation over the sticker packs, we might have a kind of optional fee and maybe when the owner of that sticker pack can specify how much a fee goes to the Croatian market automatically based on the user buys. It goes slowly rising organically as users buy it. And of course if the sticker market artist want to promote their own sticker pack they could just deposit in the Croatian market.
+
+Rachel: I think that there was also a similar suggestion made that artists when they submit their packs to the registry they could optionally donate some percentage of sales to status.
+
+Jarrad: From an implementation standpoint, going back to this as a registry discussion, from an implementation standpoint many of these look very similar. Basically it's a certain number and sending it to a certain address.
+
+31:00 - 32:00
+
+Jarrad: If we're choosing to burn then it's just zero. Zero otherwise the address be like the Eurasian market or whatever. So we can definitely defer this decision-making while not impeding implementation so much, if we just implement these calls.
```

Comment:
Eurasian market seems wrong.

@jarradh @richard-ramos @3esmit do you know? Guessing something like DAO but not sure how that'd parse